parcels/fieldset.py

```diff
@@ -109,6 +111,8 @@ def from_netcdf(cls, filenames, variables, dimensions, indices={},
                   correction for zonal velocity U near the poles.
                2. flat: No conversion, lat/lon are assumed to be in m.
         :param allow_time_extrapolation: boolean whether to allow for extrapolation
+        :param time_periodic: boolean whether to loop periodically over the time component of the dataset
+               This flags put automatically allow_time_extrapolation to False
```

Comment:
Same comments as above for `Field` docstring. I.e. change 'dataset to FieldSet and rephrase override info